### ISSUE TITLE
Re-test everything

### ISSUE THREAD

**Issue by dhouck** [dhouck/anti-creeper-grief#10]
Once gnembon/fabric-carpet#398 is fixed, re-test all the tick-perfect edge cases.

**Comment by James103** [comment]
> Once https://github.com/gnembon/fabric-carpet/issues/398 is fixed, re-test all the tick-perfect edge cases.

https://github.com/gnembon/fabric-carpet/issues/398 has been fixed for several months as of the time of writing.

**Comment by dhouck** [comment]
Yeah, I saw when it was fixed.  But I had a while where I couldnʼt work on this, and since then itʼs been lower priority for me.  Iʼll probably get back to this at some point but not for a while.

**Comment by purejosh** [comment]
Is this going to be implemented?

**Comment by dhouck** [comment]
As I said above, Iʼll probably get back to this datapack at some point, but itʼs still a low priority.  I should probably update the README and archive the repo to indicate that.
